lumen_runtime/src/lib.rs

```diff
@@ -1,4 +1,4 @@
-#![deny(warnings)]
+//#![deny(warnings)]
```

Comment:
Yeah that wasn't meant to be committed, it was temporary to make working through the compiler errors less annoying. When you have a lot of warnings, polluting the actual errors with them makes it really hard to sort the wheat from the chaff - especially since a lot of warnings tend to be addressed when the errors are fixed.